nixos/containers: create veths if only IPv6 is configured

rnhmjoj opened the pull request:
###### Motivation for this change
This fixes the failing nixos.tests.containers-ipv6 test.

###### Things done
- [x] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [x] Tested via `nixos.tests.containers-ipv6`
- [ ] Tested compilation of all pkgs that depend on this change
- [ ] Tested execution of all binary files
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [x] Assured whether relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

rnhmjoj committed:
nixos/containers: create veths if only IPv6 is configured

This fixes the failing nixos.tests.containers-ipv6 test. Thanks to andir.

andir reviewed (changes requested):
This does not fix the actual problem. It might add the addresses to the containers network but not to the external interface.

Have a look at what I quickly hacked together on Sunday:

https://github.com/andir/nixpkgs/commit/04af592789ae53d7ca184587a0349daffb61196c

That solved the issue for me on one of the servers I temporarily updated to 19.03/unstable.

rnhmjoj commented:
ping

andir committed:
nixos/release: make ipv6 tests as important as legacy IP tests

IPv6 container support broke a while ago and we didn't notice it. Making
them part of the (small) release test set should fix that. At this point
in time they should be granted the same amount of importance as the
legacy IP tests.

andir reviewed (approved):
Looks good to me. I added the tests to both test sets so we can be sure they will be noticed once they fail (again).

andir commented:
Merging since I built the `nixos-small` release successfully.
